src/profile/style.css

```diff
@@ -157,6 +163,19 @@ nav h1::before {
   border-right: 1px solid #c4c4c4;
 }
 
+.oldDiff {
+  background-color: var(--color-red);
+  padding: 4px;
+  border-radius: 2px;
+  color: var(--color-white);
+}
+
+.newDiff {
+  background-color: var(--color-green);
+  padding: 2px 4px;
+  border-radius: 2px;
+  color: var(--color-white);
+}
```

Comment:
there are 3 properties i.e padding(left and right), border-radius, and color are the same in both oldDiff and newDiff classes. Can we have a separate reusable class that has these properties, and oldDiff and newDiff classes have only the properties which are different?

Reply:
Thanks, @poonammohata for your review. I have done the changes. Kindly review.